File: sdk/go/entity/recon_notify_log.go
package entity

import (
	"time"
)

// ReconNotifyLog 对账通知日志
type ReconNotifyLog struct {
	ID           int64     `json:"id" gorm:"primaryKey"`
	OrderNo      string    `json:"orderNo"`
	SubOrderNo   string    `json:"subOrderNo"`
	MerchantId   string    `json:"merchantId"`
	NotifyUrl    string    `json:"notifyUrl"`
	NotifyStatus int       `json:"notifyStatus"` // 0:失败, 1:成功
	NotifyResult string    `json:"notifyResult"`
	CreateTime   time.Time `json:"createTime"`
	UpdateTime   time.Time `json:"updateTime"`
}

File: sdk/python/entity/recon_notify_log.py
from dataclasses import dataclass
from datetime import datetime
from typing import Optional

@dataclass
class ReconNotifyLog:
    """通知日志"""
    order_no: str
    sub_order_no: Optional[str] = None
    merchant_id: Optional[str] = None
    notify_url: Optional[str] = None
    notify_status: int = 0
    notify_result: Optional[str] = None
    
    create_time: Optional[datetime] = None
    update_time: Optional[datetime] = None
